pkg/api/http_server.go

```diff
@@ -178,6 +179,7 @@ type HTTPServer struct {
 	secretsMigrator              secrets.Migrator
 	userService                  user.Service
 	tempUserService              tempUser.Service
+	loginAttemptService          loginAttempt.Service
```

Comment:
You could technically add this later, when it's being used, but this is fine too since I think we know it'll be used ~soonish.